Car_Collision_Visulaization/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.0.0)
+project(DataVisualization VERSION 0.1.0 LANGUAGES C CXX)
+
+
+add_library(
+    Visualizer
+    src/CImg.h 
+    src/data_visualization.h
+    src/data_visualization.cpp
```

Comment:
data_visualization.h and data_visualization.cpp are not meaningful names please change them 
as i suggested before should be 
visualizer.h and visualizer.cpp